lib/MusicBrainz/Server/Edit.pm

```diff
@@ -336,6 +338,7 @@ sub TO_JSON {
             auto_edit => boolean_to_json($conditions->{auto_edit}),
         },
         created_time => datetime_to_iso8601($self->created_time),
+        display_data => $self->display_data,
```

Comment:
This broke the display of some Edit::Relationship::Delete edits (which our Selenium tests caught), so I fixed that in 1db864e1cc1dbca11d5e6d588186fe9e9a906c8c, but we'll need to test this against more edit types to be sure (especially historical ones).